tools/webassembly.py

```diff
@@ -133,7 +133,8 @@ class ExternType(IntEnum):
 
 Section = namedtuple('Section', ['type', 'size', 'offset'])
 Limits = namedtuple('Limits', ['flags', 'initial', 'maximum'])
-Import = namedtuple('Import', ['type', 'mod', 'field'])
+Import = namedtuple('Import', ['kind', 'mod', 'field'])
```

Comment:
seems like "module" would be more readable than "mod" and not really that much less to type?